<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WadSelector</class>
 <widget class="QMainWindow" name="WadSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>542</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WadSelector</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Ini file</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="IniPathBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="SetIni">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Wads path</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="WadPathBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="SetWad">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
        <property name="spacing">
         <number>54</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QListWidget" name="listWidget"/>
        </item>
        <item>
         <widget class="QPushButton" name="LoadWad">
          <property name="text">
           <string>Load</string>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>615</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>LoadWad</sender>
   <signal>clicked()</signal>
   <receiver>WadSelector</receiver>
   <slot>OnWadLoad()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SetWad</sender>
   <signal>clicked()</signal>
   <receiver>WadSelector</receiver>
   <slot>OnWadSet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>575</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>521</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SetIni</sender>
   <signal>clicked()</signal>
   <receiver>WadSelector</receiver>
   <slot>OnIniSet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnWadLoad()</slot>
  <slot>OnIniSet()</slot>
  <slot>OnWadSet()</slot>
 </slots>
</ui>
